README.md

```diff
@@ -232,11 +232,11 @@ index e93321f..7ac4fe2 100644
    return compute_metrics(logits, batch['label'])
  
 -def train_epoch(optimizer, train_ds, batch_size, epoch, rng):
-+def replicate(tree_obj, num_replicas):
-+  return jax.tree_map(lambda x: onp.array([x] * num_replicas), tree_obj)
++def replicate(tree_obj):
++  return jax.tree_map(lambda x: onp.array([x] * jax.device_count()), tree_obj)
```

Comment:
This looks like a less efficient version of jax_utils.replicate

Reply:
What do you mean exactly? How would you like me to change it?

Reply:
I've now used jax_utils.replicate, and tested the code using a google-internal colab.